src/backend/distributed/metadata/node_metadata.c

```diff
@@ -422,7 +422,7 @@ citus_disable_node(PG_FUNCTION_ARGS)
 	char *nodeName = text_to_cstring(nodeNameText);
 	WorkerNode *workerNode = ModifiableWorkerNode(nodeName, nodePort);
 	bool isActive = false;
-	bool onlyConsiderActivePlacements = false;
+	bool onlyConsiderActivePlacements = true;
```

Comment:
@marcocitus I think here we should only ignore `to-be-deleted` shards(not inactive shards etc), so I am thinking of changing these functions in that way, what do you think?

Reply:
I think that's ok. We'll probably remove the concept of inactive shard placements in a few versions anyway.

Reply:
Should we change this in citus_disable_node? Not sure, it's a slightly weird function.